aries_cloudagent/vc/data_integrity/cryptosuites/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .eddsa_jcs_2022 import EddsaJcs2022
+
+CRYPTOSUITES = {
+    "eddsa-jcs-2022": EddsaJcs2022,
+}
```

Comment:
I don't have a problem with this as is but just to note: having a static dictionary of crypto suites means this won't be extensible by plugin. Having the ability to plugin this sort of thing has been pretty handy in the past but I think it's fair to wait until there's a need before making a pluggable cryptosuite registry.

Reply:
I will consider this when adding the next cryptosuite. The intent here is that the DataIntegrity plugin invokes the cryptosuite instance given the `proof.cryptosuite` string or the `options.cryptosuite` string provided